src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -609,7 +609,7 @@ public Long getValue()
             public Long getValue()
             {
                 long min = Long.MAX_VALUE;
-                for (Metric cfGauge : allTableMetrics.get("MinPartitionSize"))
+                for (Metric cfGauge : ALL_TABLE_METRICS.get("MinPartitionSize"))
```

Comment:
nit: maybe also create constants for the metrics names? since this patch is doing some code cleanup already. 
(Maybe too much work. your call)